python/dlisio/settings.py

```diff
@@ -1,4 +1,17 @@
 from . import core
+from . import plumbing
+
+import re
+
+
+""" regex and exact matchers are frequently used by most methods on
+logicalfile. To avoid the overhead of initializing a new instance of these for
+every method-call they are cached as globals here.
+
+Although possible, these globals are not intended to be changed by the end user directly.
+"""
```

Comment:
Not sure if this is why Codacy complains, but maybe this doesn't have to be docs (as it refers to both settings), just a normal comment.
Or if we make it docs, it might have to be put under the line it refers to